skbio/io/format/embl.py

```diff
@@ -0,0 +1,1558 @@
+...      ttttcattgg gatcttcccc aagtcttaga agatgagtat ggtggtttct taaactccgg
+...      tgtaataaat gattttcgag actatacgga tctttgcttc aaggaatttg gagatagagt
+...      gaggtattgg agtactctaa atgagccatg ggtgtttagc aattctggat atgcactagg
+...      aacaaatgca ccaggtcgat gttcggcctc caacgtggcc aagcctggtg attctggaac
+...      aggaccttat atagttacac acaatcaaat tcttgctcat gcagaagctg tacatgtgta
+...      taagactaaa taccaggcat atcaaaaggg aaagataggc ataacgttgg tatctaactg
+...      gttaatgcca cttgatgata atagcatacc agatataaag gctgccgaga gatcacttga
+...      cttccaattt ggattgttta tggaacaatt aacaacagga gattattcta agagcatgcg
+...      gcgtatagtt aaaaaccgat tacctaagtt ctcaaaattc gaatcaagcc tagtgaatgg
+...      ttcatttgat tttattggta taaactatta ctcttctagt tatattagca atgccccttc
+...      acatggcaat gccaaaccca gttactcaac aaatcctatg accaatattt catttgaaaa
+...      acatgggata cccttaggtc caagggctgc ttcaatttgg atatatgttt atccatatat
+...      gtttatccaa gaggacttcg agatcttttg ttacatatta aaaataaata taacaatcct
+...      gcaattttca atcactgaaa atggtatgaa tgaattcaac gatgcaacac ttccagtaga
+...      agaagctctt ttgaatactt acagaattga ttactattac cgtcacttat actacattcg
+...      ttctgcaatc agggctggct caaatgtgaa gggtttttac gcatggtcat ttttggactg
+...      taatgaatgg tttgcaggct ttactgttcg ttttggatta aactttgtag attagaaaga
+...      tggattaaaa aggtacccta agctttctgc ccaatggtac aagaactttc tcaaaagaaa
+...      ctagctagta ttattaaaag aactttgtag tagattacag tacatcgttt gaagttgagt
+...      tggtgcacct aattaaataa aagaggttac tcttaacata tttttaggcc attcgttgtg
+...      aagttgttag gctgttattt ctattatact atgttgtagt aataagtgca ttgttgtacc
+...      agaagctatg atcataacta taggttgatc cttcatgtat cagtttgatg ttgagaatac
+...      tttgaattaa aagtcttttt ttattttttt aaaaaaaaaa aaaaaaaaaa aaaaaaaaa
+... //
+... '''
+
+
+Now we can read it as ``DNA`` object:
+
+>>> import io
+>>> from skbio import DNA, RNA, Sequence
+>>> embl = io.StringIO(embl_str)
+>>> dna_seq = DNA.read(embl)
+>>> dna_seq
+DNA
+----------------------------------------------------------------------
+Metadata:
+    'ACCESSION': 'X56734; S46826;'
+    'CROSS_REFERENCE': <class 'list'>
+    'DATE': <class 'list'>
+    'DBSOURCE': 'MD5; 1e51ca3a5450c43524b9185c236cc5cc.'
+    'DEFINITION': 'Trifolium repens mRNA for non-cyanogenic beta-
+                   glucosidase'
+    'KEYWORDS': 'beta-glucosidase.'
+    'LOCUS': <class 'dict'>
+    'REFERENCE': <class 'list'>
+    'SOURCE': <class 'dict'>
+    'VERSION': 'X56734.1'
+Interval metadata:
+    3 interval features
+Stats:
+    length: 1859
+    has gaps: False
+    has degenerates: False
+    has definites: True
+    GC-content: 35.99%
+----------------------------------------------------------------------
+0    AAACAAACCA AATATGGATT TTATTGTAGC CATATTTGCT CTGTTTGTTA TTAGCTCATT
+60   CACAATTACT TCCACAAATG CAGTTGAAGC TTCTACTCTT CTTGACATAG GTAACCTGAG
+...
+1740 AGAAGCTATG ATCATAACTA TAGGTTGATC CTTCATGTAT CAGTTTGATG TTGAGAATAC
+1800 TTTGAATTAA AAGTCTTTTT TTATTTTTTT AAAAAAAAAA AAAAAAAAAA AAAAAAAAA
+
```

Comment:
it is nice to add example of reading into `RNA` too (considering you imported `RNA` here). The goal in the doc here is to document most of the public facing functionalities. It is beneficial to demo most usage.